Index: pootle/settings.py
===================================================================
--- pootle/settings.py	(revision 16433)
+++ pootle/settings.py	(working copy)
@@ -260,3 +260,8 @@
 
     from django.template import loader
     loader.get_template = cache_templates(loader.get_template)
+
+TEST_RUNNER = 'xmlrunner.extra.djangotestrunner.run_tests'
+TEST_OUTPUT_VERBOSE = True
+TEST_OUTPUT_DESCRIPTIONS = True
+TEST_OUTPUT_DIR = 'xmlrunner'
