diff --git a/Pootle/localsettings.py b/Pootle/localsettings.py
index d17bd2b..b11b0c4 100644
--- a/Pootle/localsettings.py
+++ b/Pootle/localsettings.py
@@ -56,10 +56,10 @@ PARSE_POOL_CULL_FREQUENCY = 4
 # By default we use Django's in memory cache which is only suitable
 # for small deployments. memcached is prefered. For more info, check
 # http://docs.djangoproject.com/en/dev/topics/cache/#setting-up-the-cache
-CACHE_BACKEND = 'locmem:///?max_entries=4096&cull_frequency=5'
+#CACHE_BACKEND = 'locmem:///?max_entries=4096&cull_frequency=5'
 
 # Uncomment to use memcached for caching
-#CACHE_BACKEND = 'memcached://127.0.0.1:11211/'
+CACHE_BACKEND = 'memcached://127.0.0.1:11211/'
 
 # Using memcached to store sessions improves performance for anonymous
 # users. For more info, check
@@ -69,7 +69,7 @@ CACHE_BACKEND = 'locmem:///?max_entries=4096&cull_frequency=5'
 #SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
 
 # Uncomment this if you're using memcached as CACHE_BACKEND and running under Django 1.1
-#SESSION_ENGINE = 'django.contrib.sessions.backends.cached_db'
+SESSION_ENGINE = 'django.contrib.sessions.backends.cached_db'
 
 # To improve performance, non-logged users get cached copies of most pages.
 # This variable is the number of seconds for which a page will be reused from
